src/command.c

```diff
@@ -907,23 +920,24 @@ static int dv_convert_ipu_v0(struct device *dev, struct dmp_cmb *cmb,
 		return -EFAULT;
 
 	// prep buffer 
-	ret = get_dma_addr(dev, cmb, &cmd.wr, &wr_base_addr,
-			   &wr_buf_size);
+	ret = get_dma_addr(dev, cmb, &cmd.wr, &wr_base_addr, &wr_buf_size);
 	if (ret)
 		return ret;
-	if(cmd.use_tex) {
-		ret = get_dma_addr(dev, cmb, &cmd.tex, &tex_base_addr, &tex_buf_size);
+	if (cmd.use_tex) {
+		ret = get_dma_addr(dev, cmb, &cmd.tex,
+				&tex_base_addr, &tex_buf_size);
```

Comment:
misalignment, either break line just after the bracket or align next line just after the bracket